changelog.d/16215.bugfix

```diff
@@ -0,0 +1 @@
+Fix a bug where admin tokens stopped working with MSC3861 auth delegation was enabled.
```

Comment:
Is this _all_ admin tokens, or just the admin token reserved for the identity provider?